src/index.js

```diff
@@ -99,6 +100,33 @@ const updateDailySummaryTable = async (sprint, pointsLeft) => {
   });
 };
 
+const isWeekend = (date) => {
+  const dayOfWeek = moment(date).format("ddd");
+  return dayOfWeek === "Sat" || dayOfWeek === "Sun";
+};
+
+/**
+ * Calculates the number of weekdays from {@link start} to {@link end}
+ * @param {moment.Moment} start First day of sprint (inclusive)
+ * @param {moment.Moment} end Last day of sprint (exclusive)
```

Comment:
Why is this exclusive ah

Reply:
Preserves the current behaviour, which is if start date is 1 June and end date is 2 June, the graph will only have 2 data points 1->2 which is only one day of work

Reply:
This will work directly with your suggestion where we set the end date to be the start date of the next sprint (which if we are doing this it should be exclusive)

Reply:
Hmm I think our conclusion was that 

Sprint start: 4PM Mon
Sprint end: 10AM Mon

Say we run a cron daily at 10AM

Chart should consist of:
* x=0 - 4PM Mon (manually triggered until we automate this)
* x=1 - 10AM Tues
* x=2 - 10AM Wed
* x=3 - 10AM Thu
* x=4 - 10AM Fri
* x=5 - 10AM Mon

So i think should be inclusive?

Reply:
Yep I meant like the 24hrs of the last day of sprint is not counted, it's more of used as the end marker? Cause with this calculation there's 5 work days (0 -> 1, 1 -> 2, ... , 4 -> 5). Which corresponds to Mon, Tues, Wed, Thu, Fri. If the end is included as well then it will be calculated as 6 weekdays instead, which is wrong.

Reply:
x=0 is start, x=5 is end right?

So actually 6 is correct?

Reply:
I think actually it's supposed to be 6 weekdays if we look at it from the perspective of the x axis / number of data points on the chart. The confusion here is you're considering number of weekdays as in the weekdays that we worked, right? 

But I think that's less intuitive since the script is looking at the data from the chart's perspective. I see in the `getChartLabels` function you are also adding 1 to the number of weekdays.

I'd suggest adding 1 to numDaysInSprint in line 258, and changing this function to be number of weekdays on the chart. And in the calculation of pointsDonePerDay (line 203), do a -1 instead. In the end it's subjective how we want to do this, but I think it's more consistent to keep the perspective from the chart's. 

What do you think?

Reply:
Ohh, ok I see what you mean. Yes, that sounds more intuitive, I can change to that.

One small issue would be that I can't  just directly subtract -1 because I'll need to check if the end day is a weekday and only subtract 1 if that is true (prob just before line 203, doing something like `isWeekend(end) ? 0 : -1`).

An alternative would be that in line 259 I pass in the day before the end instead (so the function is still changed to the more intuitive `start(inclusive) to end(inclusive)`). And that at 269 it will be `isWeekendsIncluded ? numDaysInSprint : numWeekdays+1`.

Or any other suggestions?

Reply:
The last suggestion sounds good to me :)